packages/uswds-core/src/styles/mixins/general/button-unstyled.scss

```diff
@@ -10,9 +10,10 @@
   border-radius: 0;
   box-shadow: none;
   font-weight: font-weight("normal");
+  justify-content: normal;
+  text-align: left;
```

Comment:
Added text-align back in to fix a regression with Headers nav link (uses button & accordion) 

**Regression**
<img width="253" alt="image" src="https://github.com/uswds/uswds/assets/3385219/5218015f-4918-4988-8101-cd267925d646">

Reply:
Just curious, should this be defined in Header instead since that's where it's relevant?

Reply:
@mahoneycm since this line was previously removed and added back in. Moving it to header might break other uses of the mixin.

Reply:
Thanks for clarifying!